packages/pluggableWidgets/maps-native/src/Maps.editorConfig.ts

```diff
@@ -0,0 +1,26 @@
+import { Problem } from "@mendix/piw-utils-internal";
+import { MapsPreviewProps } from "../typings/MapsProps";
+
+export function check(values: MapsPreviewProps): Problem[] {
+    const errors: Problem[] = [];
+    const zoomLevels = ["world", "continent", "country", "city", "town", "streets", "building"];
+
+    const minZoomLevelIndex = zoomLevels.indexOf(values.minZoomLevel);
+    const maxZoomLevelIndex = zoomLevels.indexOf(values.maxZoomLevel);
+
+    if (minZoomLevelIndex > maxZoomLevelIndex) {
+        errors.push({
+            property: "minZoomLevel",
+            severity: "error",
+            message: "The minimum zoom level can not be smaller than the maximum zoom level."
+        });
+    } else if (minZoomLevelIndex < maxZoomLevelIndex) {
```

Comment:
If I understand this correctly, there is now an error if the `minZoomLevelIndex` is larger than `maxZoomLevelIndex` _and_ when it's smaller... so it must be equal? That can't be right.

Reply:
Whoops this isn't right. [Mehmet's comment](https://github.com/mendix/widgets-resources/pull/580#discussion_r614100121) confused me a little, he is right, but both will be catched by just one check :D